### PR TITLE
Correct after-period spacing and Type Systems link

### PR DESCRIPTION
**Pull request by dpflug**

### CODE REVIEW

File: performance.md [hunk 1]
```diff
@@ -249,14 +249,14 @@ MAX-WITH-SPEED-3
...
-can do better by declaring types.
+As you can see, the generated assembly code is much shorter (92 bytes VS 144).
+The compiler was able to perform optimizations. Yet we can do better by
+declaring types.
 
 
 ### Type Hints
 
-As mentioned in the [*Type System*][sec:type] chapter, Lisp has a relatively
+As mentioned in the [*Type System*](type.html) chapter, Lisp has a relatively
```
Comment (vindarel) [inline performance.md:259]: At the bottom of the md page, there is this ref: `[sec:type]: #type`. You didn't edit it right? Either rm or edit it (but better use the reference as it is done in this page for all the links).

**Review by vindarel — CHANGES_REQUESTED** [review]

**Comment by vindarel** [comment]
Hello, thanks for the PR.

All the after-period spacing, it's only for convenience in the markdown file, the extra spacing didn't impact the final html, right? This extra-spacing might appear again, it probably comes from someone's/my emacs M-q fill-paragraph.

**Comment by dpflug** [comment]
As for the after-period spacing, I only bothered because there were paragraphs where they were mixed. I assumed it was manually input. I can remove them if preferred.

**Comment by vindarel** [comment]
Hello, there were changes on the alive page, now there are conflicts, if you could fix them… thanks.

**Comment by dpflug** [comment]
Is there anything else needed?

**Comment by vindarel** [comment]
Thank you!

**Pull request merged**